pom.xml

```diff
@@ -105,6 +111,13 @@
                 <mainClass>in.natelev.daikondiffvictimpolluter.DaikonDiffVictimPolluter</mainClass>
             </configuration>
         </plugin>
+        
+        <plugin>
+            <groupId>edu.illinois.cs</groupId>
+            <artifactId>idflakies-maven-plugin</artifactId>
```

Comment:
Why add these two dependencies? Maybe I am missing something

Reply:
This is for running iDFlaikes